### ISSUE TITLE
Critical Bug: Crash on "Check for updates"

### ISSUE THREAD

**Issue by johnyradio** [Foundry376/Mailspring#1425]
steps:
click check for updates
mailspring crashes. 

version 1.6.0
on windows 10

**Comment by avarayr** [comment]
+1. Crashes for me as well.

**Comment by fpluquet** [comment]
+1 on Mac

**Comment by bengotow** [comment]
Yep 👍 Looks like this is the actual stack trace:

```
TypeError: Cannot read property 'createFromPath' of undefined
    at AutoUpdater.AutoUpdateManager.onUpdateNotAvailable (file:///var/folders/bb/n7t3rs157850byt_jfdcq9k80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/src/browser/autoupdate-manager.ts:177:25)
    at Object.onceWrapper (events.js:273:13)
    at AutoUpdater.emit (events.js:187:15)
    at AutoUpdater.EventEmitter.emit (domain.js:442:20)
    at AutoUpdater.topLevelDomainCallback (domain.js:121:23) {}
TypeError: Cannot read property 'createFromPath' of undefined
    at AutoUpdater.AutoUpdateManager.onUpdateNotAvailable (file:///var/folders/bb/n7t3rs157850byt_jfdcq9k80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/src/browser/autoupdate-manager.ts:177:25)
    at Object.onceWrapper (events.js:273:13)

```

Looks like we got very lucky with this one actually - this error only happens when the app is displaying the alert telling you that no update is available. The actual update flow (when there is an update) will work fine. Will get this fixed.

**Comment by bengotow** [comment]
Hey folks! This should be addressed in the latest release (1.6.3) that shipped earlier this week. The app should auto-update on it's own, or you can grab the new version from the releases page: https://github.com/Foundry376/Mailspring/releases/. Thanks for filing this - enjoy the update!
